game/scripts/vscripts/abilities/swiper/boss_swiper_swipe.lua

```diff
@@ -0,0 +1,80 @@
+function DebugRange(caster, range, ability)
+	for i=1,8 do
+		local point = caster:GetAbsOrigin() + PointOnCircle(range, 360 / 16 * i)
+		DebugDrawSphere(RotatePosition(caster:GetAbsOrigin(), caster:GetAngles() + QAngle(0,-90,0), point), Vector(255,0,0), 255, 32, true, ability:GetCastPoint())
+	end
+end
+
+--------------------------------------------------------------------------------
+
+function FindUnitsInCone(position, coneDirection, coneLength, coneWidth, teamNumber, teamFilter, typeFilter, flagFilter, order)
+	local units = FindUnitsInRadius(teamNumber, position, nil, coneLength, teamFilter, typeFilter, flagFilter, order, false)
+
+	coneDirection = coneDirection:Normalized()
+
+	local output = {}
+
+	for _, unit in pairs(units) do
+		local direction = (unit:GetAbsOrigin() - position):Normalized()
+		if direction:Dot(coneDirection) >= math.cos(coneWidth/2) then
+			table.insert(output, unit)
+		end
+	end
+
+	return output
+end
+
+--------------------------------------------------------------------------------
+
+function PointOnCircle(radius, angle)
```

Comment:
avoid global function, this could conflict with other functions if we starts making them global.
either move to a helper class on components or put it on the ability/modifier class